server/controllers/work/deleteWork.js

```diff
@@ -0,0 +1,22 @@
+const { Work } = require('../../database/models');
+const { customError } = require('../errors');
+const { paramsValidation } = require('../../utils/validation');
+
+const deleteWork = async (req, res, next) => {
+  try {
+    const { id } = await paramsValidation.validateAsync(req.params);
+    const work = await Work.findByPk(id);
+    if (!work) {
+      throw customError('work does not exist', 400);
+    }
+    await work.destroy({ where: { id } });
```

Comment:
you need to check first if the user who send the request the same how own the work